### ISSUE TITLE
Calling SetGlobalParam from Volume by code not working 

### ISSUE THREAD

**Issue by OmarVector** [guycalledfrank/bakery-issues#141]
I've baked Volume, I did instantiate it.

Now I want to set its effect to affect the target shaders, If I called SetGlobalParam, nothing happen. however, from inspector, if I set isGlobal Off, and then turn it on again, the baked volume start to render .

any reason why this is happening?

I cannot get the baked volume to run based on this steps, even if I triggered On and Off by code, has no effect either.

**Comment by guycalledfrank** [comment]
This is weird, since UI is calling the exact same function:

![image](https://user-images.githubusercontent.com/7008423/192143673-e7a3f9be-647b-4e9d-acf0-03b39fb078be.png)

Wild guess:
a) Try calling UpdateBounds() before SetGlobalParams()?
b) Something overrides SetGlobalParams() after your code is exectuted? Like, another global volume in the scene? When do you call it (editor/playmode/awake/start/etc)?

**Comment by OmarVector** [comment]
Yes, I checked, and then I found that when I call the method on multiply Volumes, only one of them is applied which is the last one, in my case, the last one is far from my source and I didnt spot it.

Now I understand that we can only set global to single volume . and must use volume trigger component for further interaction.
